patroni/postgresql.py

```diff
@@ -105,6 +106,13 @@ def can_rewind(self):
                 data.get('Data page checksum version', '0') != '0'
         return False
 
+    @property
+    def sysid(self):
+        if not self._sysid:
+            data = self.controldata()
+            self._sysid = data and data.get('Database system identifier', None)
```

Comment:
self._sysid would be assigned to {} (empty dict) if the data is an empty dict...